cli/src/commands/jobs.rs

```diff
@@ -1,5 +1,6 @@
 use std::io;
 use std::str::FromStr;
+use std::path::Path;
```

Comment:
`rustfmt` wants to sort these alphabetically.

```suggestion
use std::path::Path;
use std::str::FromStr;
```